Dockerfile

```diff
@@ -10,4 +10,4 @@ COPY . /usr/src/app
 RUN pip install --upgrade pip
 RUN pip install -r requirements.txt
 
-CMD gunicorn --bind 0.0.0.0:${RDF_DIFFER_API_PORT} rdf_differ.entrypoints.flask_app:app
\ No newline at end of file
+CMD gunicorn --bind 0.0.0.0:${RDF_DIFFER_API_PORT} rdf_differ.entrypoints.api.run:app
```

Comment:
I wonder how to run more than one service in a single Docker container.

Reply:
it is certainly possible: [docker documentation](https://docs.docker.com/config/containers/multi-service_container/), [medium article](https://towardsdatascience.com/run-multiple-services-in-single-docker-container-using-supervisor-b2ed53e3d1c0)
but the examples that I saw seem to expose only one service to the "outside" (a **django** service and **redis**) and it I think that the configuration of the endpoints of having both the `api` and the `ui` on the same docker might become problematic (`api.meaningfy.co` and `meaningfy.co`)